PrepInsta Top 100 Codes/Getting Started/Check_Palindrome.java

```diff
@@ -0,0 +1,78 @@
+//Wrtite a java program to check whether an entered number is a palindrom or not
+
+//Importing scanner class inorder to use nextInt() method for taking input from user
+import java.util.Scanner;
+
+/*
+* We are given an integer input as the number, the objective is to check whether or not the given number is a palindrome.
+* To do so, we’ll first reverse the string input using loops and check if it matches the original number.
+* If it matched then the given input number is a palindrome number
+*
+* Example-1
+* -------
+* Input : 121
+* Output : This is a palindrome number
+
+* Example-2
+* -------
+* Input : 369
+* Output : This is not a palindrome number
+*
+* */
+public class Check_Palindrome {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter a number:");
+        int number = input.nextInt();// Taking input from the user
+        // If isPalindrome() method returns true then the number is palindrome and if it
+        // returns false then it is not
```

Comment:
Remove the comment line 28 and merge it to the line number 27 and make it single comment line.